### PR TITLE
Fabo/remove old tx code

### PR DESCRIPTION
**Pull request by faboweb**
Deprecate old tx code and do a bunch of cleanups

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -20,6 +20,5 @@ export default {
 
   graphqlHost: graphql,
 
-  e2e: process.env.VUE_APP_E2E || false,
-  enableTxAPI: process.env.VUE_APP_ENABLE_TX_API === "true" || false
+  e2e: process.env.VUE_APP_E2E || false
```
Comment (Bitcoinera) [inline config.js]: 👀

File: src/ActionModal/components/ActionModal.vue [hunk 2]
```diff
@@ -480,21 +479,6 @@ export default {
...
-        chainId: this.network.chain_id,
-        connected: this.connected,
-        userAddress: this.session.address,
-        rewards: this.rewards,
-        totalRewards: this.overview.totalRewards,
-        bondDenom: this.network.stakingDenom,
-        isExtensionAccount: this.isExtensionAccount,
-        account: this.overview.accountInformation
-      }
-    },
```
Comment (Bitcoinera) [inline src/ActionModal/components/ActionModal.vue]: I guess this deprecates your prior PR #3628 😅

File: src/ActionModal/utils/ActionManager.js [hunk 3]
```diff
@@ -4,12 +4,7 @@ import transaction from "./transactionTypes"
 import { uatoms } from "scripts/num"
 import { toMicroDenom } from "src/scripts/common"
 import { getGraphqlHost } from "scripts/url"
-import {
-  getMessage,
-  getMultiMessage,
-  getTransactionSigner,
-  transformMessage
-} from "./MessageConstructor.js"
+import { signedTransactionCreator, getMessage } from "./MessageConstructor.js"
```
Comment (Bitcoinera) [inline src/ActionModal/utils/ActionManager.js]: 👀  OK, so it doesn't deprecate #3268 then. Here is our friend `signedTransactionCreator` again (whose naming I don't like 😛 )

File: src/ActionModal/utils/ActionManager.js [hunk 4]
```diff
@@ -19,56 +14,6 @@ const txFetchOptions = {
...
-    if (!this.context) {
-      throw Error("This modal has no context.")
-    }
-    this.txProps = transactionProperties
-
-    this.messageTypeCheck(type)
-    this.messageType = type
-    this.message = await getMessage(type, transactionProperties, this.context)
-  }
-
```
Comment (Bitcoinera) [inline src/ActionModal/utils/ActionManager.js]: 👀

File: src/ActionModal/utils/ActionManager.js [hunk 5]
```diff
@@ -163,48 +76,40 @@ export default class ActionManager {
...
 
-    const createSignedTransaction = await getTransactionSigner(context)
+    const createSignedTransaction = await signedTransactionCreator(networkId)
     const signedMessage = await createSignedTransaction(
       messageMetadata,
       txMessages,
       signer,
-      context.chainId,
-      context.account.accountNumber,
-      context.account.sequence
```
Comment (Bitcoinera) [inline src/ActionModal/utils/ActionManager.js]: So, I'm trying to get this PR. OK, the `Txservice` can now go to hell. But what's up with context? Why is it of no use now with transaction abstraction? I really want to understand this PR

**Comment by Bitcoinera** [comment]
My question here would be, with this one in place, do we still need the `network_type` and the `getCosmosLocalSigner`, `getCosmosLedgerSigner` and so on?

I guess so?

File: config.js (hunk 1 above)
Comment (faboweb) [reply config.js]: why the eyes? ^^

File: src/ActionModal/components/ActionModal.vue (hunk 2 above)
Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue]: Directly into sendTxApi and simulateTxAPi

File: src/ActionModal/utils/ActionManager.js (hunk 3 above)
Comment (faboweb) [reply src/ActionModal/utils/ActionManager.js]: Why don't you like the name? I think the link you posted is wrong

File: src/ActionModal/utils/ActionManager.js (hunk 5 above)
Comment (faboweb) [reply src/ActionModal/utils/ActionManager.js]: This PR is really just removing code and simplifying the existing code by removing some abstractions

**Pull request merged**